content/posts/2020-03-21-getting-r-to-print-english-error-messages_es.Rmarkdown

```diff
@@ -0,0 +1,67 @@
+---
+title: Cómo hacer que R devuelva errores en inglés
+author: Thomas Neitmann
+date: '2020-03-21'
+slug: getting-r-to-print-english-error-messages
+categories:
+  - R
+  - bytesized
+tags:
+  - base
+toc: no
+images: ~
+---
+
+Si vives en un país de habla no inglesa, como es mi caso, segúramente te aparecerán mensajes de error en tu idioma cuando las cosas van mal en `R`.
+
+```{r, eval=FALSE}
+1 + "r"
+```
+
+```
+## Error in 1 + "r": nicht numerisches Argument für binären Operator
+```
```

Comment:
Could you use the Spanish error message here?

Reply:
I live in Germany, but I will try to generate a Spanish error following your post ;)